srv/salt/ceph/tuned/mgr/default.sls

```diff
@@ -0,0 +1,19 @@
+/usr/lib/tuned/ses-mgr/tuned.conf:
+  file.managed:
+    - source: salt://ceph/tuned/ses-mgr/tuned.conf
+    - makedirs: True
+    - user: root
+    - group: root
+    - mode: 644
+
+apply tuned ses mgr:
+  cmd.run:
```

Comment:
Is there a reason for using so many "cmd.run" ? 

1. I have in mind that cmd.run should be replaced with cmd.shell (not sure if that is true)
2. cmd.run does not have much error handling AFAIK - and due to that we should use